activiti-cloud-starter-query/src/test/java/org/activiti/cloud/services/query/QueryProcessInstancesIT.java

```diff
@@ -67,84 +65,87 @@ public void shouldGetAvailableProcInstances() throws Exception {
         //given
         // a completed process
         producer.send(aProcessStartedEvent(System.currentTimeMillis(),
-                                           "10",
-                                           "defId",
-                                           "15"));
+                "10",
+                "defId",
+                "15"));
         producer.send(aProcessCompletedEvent(System.currentTimeMillis(),
-                                             "10",
-                                             "defId",
-                                             "15"));
+                "10",
+                "defId",
+                "15"));
 
         // a running process
         producer.send(aProcessStartedEvent(System.currentTimeMillis(),
-                                           "11",
-                                           "defId",
-                                           "16"));
-
-        await().untilAsserted(() -> {
-
-            //when
-            ResponseEntity<PagedResources<ProcessInstance>> responseEntity = executeRequestGetProcInstances();
-
-            //then
-            assertThat(responseEntity).isNotNull();
-            assertThat(responseEntity.getStatusCode()).isEqualTo(HttpStatus.OK);
-
-            Collection<ProcessInstance> processInstances = responseEntity.getBody().getContent();
-            assertThat(processInstances)
-                    .extracting(ProcessInstance::getProcessInstanceId,
-                                ProcessInstance::getStatus)
-                    .contains(tuple("15",
-                                    "COMPLETED"),
-                              tuple("16",
-                                    "RUNNING"));
-        });
+                "11",
+                "defId",
+                "16"));
+
+        waitForMessage();
```

Comment:
Is it intentional that you are coming back to `waitForMessage()` instead of ` await().untilAsserted()` or is it a side effect of a merge conflict?